mava/adders/reverb/transition.py

```diff
@@ -83,7 +83,9 @@ def __init__(
         client: reverb.Client,
         n_step: int,
         discount: float,
+        *,
```

Comment:
So I saw online: "The * indicates the end of the positional arguments. Every argument after that can only be specified by keyword." Just wondering why we are being explicit about it?

Reply:
My understanding of `*,` is that it is useful when they are multiple arguments and you want to split "common" arguments from "less common" arguments. 

The arguments before `*,` can be provided positionally and that usually makes sense for parms that are common to a function call. The params after `*,` , are less common and would be difficult to keep track of their positional spot since there are a lot of them and so are keyword args.